src/server/notifications/email/systemMail/userInvitivation.tsx

```diff
@@ -0,0 +1,22 @@
+import { generateJWT } from '@/jwt/jwt'
+import type { UserFiltered } from '@/server/users/Types'
+import 'server-only'
+import { userInvitationExpiration } from './ConfigVars'
+import { sendSystemMail } from '@/server/notifications/email/send'
+import { UserInvitationTemplate } from '@/server/notifications/email/templates/userInvitation'
+
+
+export async function sendUserInvitationEmail(user: UserFiltered) {
+    const jwt = generateJWT('verifyemail', {
+        sub: user.id,
+        email: user.email,
+    }, userInvitationExpiration)
+
+    const link = `${process.env.NEXTAUTH_URL}/register?token=${jwt}`
```

Comment:
It feels a bit iffy using an environment variable created for next auth elswhere in our project. Especially since you've used `DOMAIN` elsewhere, for example in `dispatch.ts`. Logically we should be using `DOMAIN` and set it to https://localhost:80. Then we could set `NEXTAUTH_URL=DOMAIN` to simplify things. However, I guess it might look a bit weird having localhost plastered all over the website instead of omega.ntnu.no, but that might not be a bad thing because then we could easily spot hard coded URLs. Another option would be to add another environment variable so we could differentiate between the actual link url and the domain which should appear for visual thing only.